checker/src/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -633,6 +635,24 @@ private void computeFieldAccessType(AnnotatedTypeMirror type,
         }
     }
 
+    /**
+     * Returns the annotations explicitly written on a constructor result.
+     * Callers should check that {@code constructorDeclaration} is in fact a declaration
+     * of a constructor.
+     *
+     * @param constructorDeclaration Declaration tree of constructor
+     * @return set of annotations explicit on the resulting type of the constructor
+     */
+    public Set<AnnotationMirror> getExplicitAnnotationsOnConstructorResult(MethodTree constructorDeclaration) {
```

Comment:
Should this method be moved to `AnnotationUtils`? Do we have `getExplicitAnnotationsOnXXX` for other XXX?
It looks like the method could be static and doesn't seem initialization specific.